AK/ByteBuffer.h

```diff
@@ -92,6 +92,21 @@ class ByteBuffer {
         return copy(bytes.data(), bytes.size());
     }
 
+    [[nodiscard]] static ErrorOr<ByteBuffer> xor_buffers(ReadonlyBytes first, ReadonlyBytes second)
+    {
+        if (first.size() != second.size())
+            return Error::from_string_literal("Buffers must be of equal size");
```

Comment:
It looks like this is causing a build error. I think it's because `ByteBuffer` is used in the Kernel, so `Error::from_errno()` should be used here instead.